js/core/utils/dom.js

```diff
@@ -198,6 +198,57 @@ setPublicElementWrapper(function(element) {
     return element && element.get(0);
 });
 
+var calculateLimitHeight = function(value, container, offset, defaultValue) {
+    if(!value) {
+        return defaultValue;
+    }
+
+    if(value === "auto" || value === "none" || value === "inherit" || value === "initial") {
+        return offset ? defaultValue : value;
+    }
+
+    if(typeof value === "string") {
+        if(value.indexOf("px") > 0) {
+            value = parseInt(value.replace("px", ""));
+        } else if(value.indexOf("%") > 0) {
+            value = parseInt(value.replace("%", "")) * $(container).height() / 100;
+        } else if(!isNaN(value)) {
```

Comment:
`!isNaN(value)` do we really need this check?

Reply:
we need to parse string where the whole value is a number, not "10mm" or something else